src/scss/partials/_logo.scss

```diff
@@ -33,12 +33,14 @@ $logo-color--transparent    : $white;
             width: 165px;
             height: 32px;
         }
-
-        path {
-            fill: $logo-color;
-
-            .c-header--transparent & {
-                fill: $logo-color--transparent;
+        // don't apply this to the Menulog logo, as it has multiple fill values built in
+        @if ($theme != 'ml') {
```

Comment:
Would be better to just do `if ($theme = 'je')` if this should just be applied to the Just Eat logo.

Reply:
It leaves a little bit flexibility. Maybe in future we may use same rule for other brands :)